test/controllers/samples_controller_test.rb

```diff
@@ -214,10 +214,13 @@ class SamplesControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
 
     results = @response.parsed_body["pipeline_stage_results"]
```

Comment:
Arent these tests covered in spec? If so, you do not need to continue to update them. Should actually remove them.